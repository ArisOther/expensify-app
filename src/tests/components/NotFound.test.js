import React from 'react'
import {shallow} from 'enzyme'
import NotFoundPage from '../../component/NotFound'
//import expenses from '../fixtures/expenses'

test('harus bisa merender NotFoundPage',()=>{
    const wrapper = shallow(<NotFoundPage/>)
    expect(wrapper).toMatchSnapshot()
})