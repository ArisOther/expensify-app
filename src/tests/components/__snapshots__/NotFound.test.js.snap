// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`harus bisa merender NotFoundPage 1`] = `
<div>
  404! 
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
